docs/guides/react-components/install-plain-java.md

```diff
@@ -8,8 +8,169 @@ For this installation, you must have:
 - OpenJDK 21+ locally installed
 - Camunda `8.6.0-alpha2` or later
 
+### Download and configure Elasticsearch
+
+1. Download [Elasticsearch 8.9.2](https://www.elastic.co/downloads/past-releases/elasticsearch-8-9-2).
+2. Disable Elasticsearch's security packages by setting the `xpack.security.*` configuration options to `false` in `ELASTICSEARCH_HOME/config/elasticsearch.yml`.
+3. Start Elasticsearch by running `ELASTICSEARCH_HOME/bin/elasticsearch` (or `ELASTICSEARCH_HOME\bin\elasticsearch.bat` on Windows).
+
+### Download and configure Camunda
+
 1. Download the [latest release artifact](https://github.com/camunda/camunda/releases), starting with [8.6.0-alpha2](https://github.com/camunda/camunda/releases/tag/8.6.0-alpha2).
```

Comment:
Running alpha2 seemed to result in a version mismatch for me, but running alpha3 went fine:

`io.camunda.operate.schema.SchemaStartup - Schema startup failed: Error(s) in index schema: Newer version(s) for operation (8.4.0) already exists: [8.4.1]`

Reply:
@romansmirnov thoughts on this?

Reply:
Assumption: The same ES instance has been used, but first, alpha3 was started, creating an index in version 8.4.1, and then alpha2 was started, expecting the index to be in version 8.4.0.

Can you please confirm my understanding? If so, I don't recommend using the ES instance and switching from a new C8 version to an old C8 version.

Reply:
@conceptualshark I don't know if we should expect people using the same ES instance (purposefully or accidentally). Can I give you a follow up here to maybe add a small troubleshooting section or think more about this experience for new users?

Reply:
The only thing I can think of that would have impacted this is having started C8SM Run on this machine (it was not running at the time). When I tested this PR, I ran ES + alpha2, failed with the above error, stopped both, then reinstalled and ran ES + alpha3, so I am not sure why an index (or even ES instance) would have been maintained. 

Unfortunately, I can't add anything for resolving it, because it still doesn't work on my machine (I had to run it in a VM).